pipelines/common/src/test/resources/sql-on-fhir-v2-tests/basic.json

```diff
@@ -2,9 +2,9 @@
   "title": "basic",
   "description": "basic view definition",
   "fhirVersion": [
-    "5.0",
-    "4.0",
-    "3.0"
+    "5.0.0",
```

Comment:
I see that the file in the SQL-on-FHIR-V2 repo over [here](https://github.com/FHIR/sql-on-fhir-v2/blob/master/tests/basic.json) does not contain any fhir versions, can you please point me to the correct link.

Reply:
Thats great @bashir2